src/revivalpmmp/pureentities/PureEntities.php

```diff
@@ -115,9 +113,10 @@ class PureEntities extends PluginBase implements CommandExecutor {
      * @return PureEntities the current instance of the plugin main class
      */
     public static function getInstance(): PureEntities {
-        return self::$instance;
+        return PureEntities::$instance;
```

Comment:
Why not use `self` instead of the class name?

Reply:
again, not me - this was in master when I pulled and merged.